src/plugins/data/common/search/aggs/agg_config.ts

```diff
@@ -92,6 +93,7 @@ export class AggConfig {
   public parent?: IAggConfigs;
   public brandNew?: boolean;
   public schema?: string;
+  public buckets?: TimeBuckets;
```

Comment:
why was this added ?

Reply:
If I remember right, I've done it for this case:
https://github.com/elastic/kibana/blob/4bd6b39d768d10d4d85033860b467260291a6051/src/plugins/vis_types/vislib/public/to_ast.ts#L50

That field exists at the`AggConfig` instance.